npm/react/package.json

```diff
@@ -15,7 +15,7 @@
   "dependencies": {
     "@babel/plugin-transform-modules-commonjs": "7.10.4",
     "@cypress/code-coverage": "3.8.1",
```

Comment:
Seems only some packages had their versions updated - should they all be `0.0.0-development`? Why are some `*` and some not?